nemo_text_processing/text_normalization/en/taggers/electronic.py

```diff
@@ -110,12 +110,15 @@ def __init__(self, cardinal: GraphFst, deterministic: bool = True):
         # abc.com, abc.com/123-sm
         # when only domain, make sure it starts and end with NEMO_ALPHA
         graph |= (
-            pynutil.insert("domain: \"")
+            pynutil.insert('domain: "')
             + pynini.compose(
-                NEMO_ALPHA + pynini.closure(NEMO_NOT_SPACE) + accepted_common_domains + pynini.closure(NEMO_NOT_SPACE),
+                NEMO_ALPHA
+                + pynini.closure(NEMO_NOT_SPACE)
+                + accepted_common_domains
+                + pynini.closure(pynini.difference(NEMO_NOT_SPACE, pynini.accep("."))),
```

Comment:
Can you check this with nested subdomains? e.g. `.gradcenter.cuny.edu`

Reply:
>>> python3 normalize.py --text="www.gradcenter.cuny.edu" --lang="en" --verbose
 NeMo-text-processing :: INFO     :: Creating ClassifyFst grammars.
 NeMo-text-processing :: DEBUG    :: tokens { electronic { protocol: "WWW dot " domain: "GRADCENTER.CUNY.edu" } }
========================================
WWW dot gradcenter dot cuny dot edu
========================================
 NeMo-text-processing :: INFO     :: Execution time: 0.03 sec

Reply:
Wunderbar.